conda_forge_tick/migrators/version.py

```diff
@@ -691,7 +691,8 @@ def pr_body(self, feedstock_ctx: FeedstockContext) -> str:
             hint = f"\n\nDependency Analysis\n--------------------\n\n"
             hint += (
                 "We couldn't run dependency analysis due to an error. Make sure the "
-                "recipe and the source code download URL are correct.\n"
+                "source code download URL is correct and any source code patches "
+                "apply correctly.\n"
```

Comment:
How can a person do that?

Reply:
They can edit the recipe and any patches so they apply correctly.

Reply:
Isn't that a given since the CI are run? I think message would only confuse a user.

Reply:
I don't have strong opinions here FWIW. I was trying to add a hint on why things might have failed and what one might do about it.

Reply:
I think a message saying that this is due to a bot limitation is enough.

Reply:
Ah. The source code download URL being correct has to be true or the PR would not have worked. The CI will fail because the patches won't apply which is why they got this message. 

Would you rather we simply take it out?

Reply:
I think a message saying that this a bot limitation and help is welcome is good to have.